distro/configuration/ampathforms/tb_case_enrollment_v1.0.json

```diff
@@ -0,0 +1,907 @@
+                  {
+                    "concept": "160050AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Individualized treatment"
+                  },
+                  {
+                    "concept": "160048AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "empiric treatment"
+                  }
+                ]
+              },
+              "validators": []
+            },
+            {
+              "label": "MDR regimen",
+              "type": "obs",
+              "required": false,
+              "id": "drRegimen",
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "159909AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "conceptMappings": [
+                  {
+                    "relationship": "NARROWER-THAN",
+                    "type": "AMPATH",
+                    "value": "2161"
+                  },
+                  {
+                    "relationship": "SAME-AS",
+                    "type": "CIEL",
+                    "value": "159909"
+                  }
+                ],
+                "answers": [
+                  {
+                    "concept": "128ea82a-1301-4ec0-b9de-9f98c55c6bb5",
+                    "label": "6 (H)REZ/Lfx"
+                  },
+                  {
+                    "concept": "82bd6821-1897-491b-9597-e7ba9d33e208",
+                    "label": "6 Bdq/Cfz/Lfx/Cs/Lzd 12 Cfz/Lfx/Cs"
+                  },
+                  {
+                    "concept": "c70c1789-bbf4-4a0f-8727-0bc3e310c19a",
+                    "label": "6 Bdq/DIm/Lzd/Cfz/Cs/ 14 DIm/Cfz/Cs"
+                  },
+                  {
+                    "concept": "271209f1-068a-449e-acc0-3c18c5bfcf75",
+                    "label": "6 Bdq/Dlm/Lzd/Cfz/Cs 14 Dlm/Cfz/Cs/Z"
+                  },
+                  {
+                    "concept": "4b630a0c-7bce-44d9-8af1-dab602edc6dc",
+                    "label": "6 Bdq/Dlm/Lzd/Cfz/Cs/Z 14 Dlm/Cfz/Cs/Z"
+                  },
+                  {
+                    "concept": "60052d37-3e61-460b-b887-d672093b0bb1",
+                    "label": "6 Dlm/Lzd/Lfx/Cfz/Cs 12 Lfx/Cfz/Cs"
+                  },
+                  {
+                    "concept": "4c852067-84c0-4dd9-8eaa-77d3f40f2c20",
+                    "label": "6 Lzd/Mfx/Cfz/Cs 12 Mfx/Cfz/Cs"
+                  },
+                  {
+                    "concept": "ad311897-b20e-4009-ac9a-e5a322498bee",
+                    "label": "6 Mfx/Cfz/Cs/Lzd 12 Mfx/Cfz/Cs"
+                  },
+                  {
+                    "concept": "839a3a51-07c3-4316-8efc-429b6471d314",
+                    "label": "9 (H)REZ/Lfx"
+                  }
+                ]
+              },
+              "validators": []
+            },
+            {
+              "label": "MDR regimen - Free text",
+              "type": "obs",
+              "required": false,
+              "id": "drRegimen",
```

Comment:
@WamathagaUCSF This question has the same question id as `MDR regimen` and as such will display the concept UUID of selected answers for the `MDR regimen`. Is this a duplicate question or?